src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.cpp

```diff
@@ -1946,6 +1946,15 @@ namespace UiaOperationAbstraction
         ToRemote();
     }
 
+    UiaVariant::UiaVariant(UiaElement value) :

```

Comment:
Thanks so much for fixing this! As I'm sure you've found out, there's a number of supported scenarios we just haven't had the chance to port to UiaOperationAbstraction yet. :)